src/capymoa/anomaly/half_space_trees.py

```diff
@@ -0,0 +1,63 @@
+from capymoa.base import (
+    MOAClassifier,
+)
+
+from moa.classifiers.oneclass import HSTrees as _MOA_HSTrees
+
+
+class HalfSpaceTrees(MOAClassifier):
```

Comment:
Please add a MOAOutlier generic wrapper for anomaly detection methods and an abstract class Outlier. These will be similar, but different from the `Classifier` and `MOAClassifier`